db/schema.rb

```diff
@@ -440,7 +440,6 @@
     t.string "ssn_signature", limit: 64
     t.text "encrypted_pii_recovery"
     t.integer "deactivation_reason"
-    t.boolean "phone_confirmed", default: false, null: false
```

Comment:
I beleive we can also safely removed the `ignored_column` line in the same PR as this, since when this code boots up, the column will have already been removed